docs/styleguides/python.md

```diff
@@ -197,11 +197,11 @@ from test_util.helpers import (CI_CREDENTIALS,
 
 * TODO Comment Format
     * Follow the rules on [when to write TODO comments](https://google.github.io/styleguide/pyguide.html?showone=TODO_Comments#TODO_Comments)
-    * Format required will be `TODO (active-developer) [JIRA] - Brief one line description`
+    * Format required will be `#TODO (active-developer) JIRA_ID: Brief one line description`
 
     ```
-    # TODO: (skumaran) DCOS_OSS-952 - Create a flake8 plugin to enforce TODO format.
-    ````
+    #TODO (skumaran) DCOS_OSS-952: Create a flake8 plugin to enforce TODO format.
```

Comment:
> Then why not just a JIRA?

Because, it captures the person responsible for the `TODO` inline and it helps both when writing the TODO and for another developer coming across the TODO.

> Jira owner can change, developer can move to a different project/team.

Or a developer might leave the project. 

In that situation, it might be a good idea to evaluate the TODO and take one of the necessary actions, like a) kill it b) re-assign to a different active developer.